index.rst

```diff
@@ -302,107 +305,152 @@ Is this case, we could have something like:
+    - It is not possible to load this file using the `configurationToApply` attribute
+    - Between this file and the ``_init.yaml`` file, **the configuration must be fully defined**.
 
-#.  On the configuration repository there shall be a ``_base.yaml`` file defining all the base configuration values (we use "base" instead of "default").
+#.  An additional configuration file provides overrides for the configuration parameters set by the previous files.
 
-    - See this :download:`example _base.yaml <_static/_base.yaml>` for the ATSpectrograph CSC.
+    - See this :download:`configuration parameter override example file <_static/ATSpectrograph_example_config.yaml>` for the ATSpectrograph CSC.
+    - This file is the third configuration file to get loaded by the CSC and can override any previously declared values.
+    - These files are loaded using the `configurationToApply` attribute in the ``start`` command
+    - These are not expected to be required as part of regular operations and are meant to be used when a non-standard configuration is required
+    - If an override configuration file is also site specific, then a prefix should be added indicating which site it belongs with (e.g. ``summit_reduced_stage_travel.yaml``)
 
 #.  The labels file (e.g. ``_labels.yaml``) shall continue to exist with the same format and purpose.
-#.  Additional configuration files can provide new values for individual configuration parameters.
 
-    - See this :download:`example configuration file <_static/ATSpectrograph_example_config.yaml>` for the ATSpectrograph CSC.
+        - No label shall exist having the name ``default``
+
+#.  If a CSC receives a ``start`` command with an empty ``configuration`` (see :ref:`renaming proposal <section-renaming>`) attribute, it shall load the values in ``_init.yaml`` then the site-specific file (e.g. ``_summit.yaml``).
+#.  If a CSC receives a ``start`` command with a ``configuration`` attribute equal to a label in ``_labels.yaml``, it shall load the values in ``_init.yaml``, then then the site-specific file (e.g. ``_summit.yaml``), and lastly the override file.
 
-#.  If a CSC receives a ``start`` command with an empty ``configuration`` (see :ref:`renaming proposal <section-renaming>`) attribute, it shall load the values in ``_base.yaml``.
-#.  If a CSC receives a ``start`` command with a ``configuration`` attribute equal to a label in ``_labels.yaml``, it shall load the values in ``_base.yaml`` first and override those values defined in the mapped configuration file.
-#.  The name ``default`` shall not be used for labels.
 #.  All valid configurations shall have a label to be loaded by the CSC.
-    Configurations cannot be loaded by filename.
-#.  The configuration repository should be kept clean of configurations used for unit testing or for different purposes (e.g. test stand configurations).
-    Configurations tailored for test stand can be kept in different branches/tags.
+
```

Comment:
Why do you insist that every override file have a label? Right now salobj allows loading `foo.yaml` and I would prefer to continue to allow that. In fact I would argue we can do away with labels entirely. Override files will probably be rare, and hopefully short-lived, since the site-specific configuration should usually suffice.

Requiring a label for every override file greatly increases the amount of effort required to add an override file. One has to add the file itself, then add an entry to the labels dict, and then be sure to spell the file name correctly in the labels dict. There are too many places where that can go wrong.

Similarly, one has to remember to edit the labels dict when one retires (gets rid of) an override file.

Reply:
This was more done for heritage sake... but we're getting rid of labels! so this will be moot upon the subsequent revision.